html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label for="email">Email</label>
```

Comment:
This `for` attribute here is correct, but the form control element it is paired with needs a matching `id` attribute with the same value.